tools/report-converter/codechecker_report_converter/analyzer_result.py

```diff
@@ -72,6 +72,14 @@ def _add_report_hash(self, plist_objs):
 
     def _get_analyzer_result_file_content(self, result_file):
```

Comment:
I had some comments on this function in #2649.

Reply:
@bruntib I will check it and fix your comments on that pull request.